### DISCUSSION TITLE
How to view number of times that one file was viewed / downloaded?

### DISCUSSION THREAD

**Discussion by antoniovandre2** [community/community#69016] [Repositories]
### Select Topic Area

Question

### Body

In order to check for updates, whenever someone runs my software, the version file is consulted. I would like to know how many times it has been accessed.

**Comment by HarshanaJayasekara** [comment]
- If you want to track downloads of specific files, you can consider creating releases within your GitHub repository.
When you create a release, you can attach binary files (such as software packages) to it. GitHub will then provide download statistics for those files.
To create a release, go to your repository on GitHub, click on the "Releases" tab, and then click the "Create a new release" button.
You can attach files to the release, and GitHub will track the number of times each file has been downloaded.
